v4/common/src/main/java/exchange/dydx/trading/common/container/ToastContainer.kt

```diff
@@ -0,0 +1,177 @@
+package exchange.dydx.trading.common.container
```

Comment:
Can we move this to PlatformUI, and make it a replacement of the PlatformInfo class?